package.json

```diff
@@ -29,7 +31,9 @@
     "chalk": "^5.2.0",
     "execa": "^7.1.1",
     "fastify": "^4.17.0",
-    "pino-pretty": "^10.0.0"
+    "patch-package": "^7.0.0",
+    "pino-pretty": "^10.0.0",
+    "@fastify/reply-from": "9.1.0"
```

Comment:
pinpoint @fastify/reply-from instead of using `override` as it requires a certain npm version, this is a hack that relies on how npm optimize dependency works